setup.md

```diff
@@ -0,0 +1,23 @@
+Initial setup
+=============
+```bash
+git clone --recurse-submodules git@github.com:gtfierro/hoddb.git
+cd hoddb
+```
+
+Install go
+==========
+* [Install go](https://golang.org/doc/install)
+* source setup.sh:
+```bash
+. /setup.sh
```

Comment:
If I remember correctly, dependencies should be taken care of by the existing Go tooling. Just run

```
$ go get
$ go build
```

from inside the project directory. I'd prefer to handle dependencies that way, as it is standard for most Go projects.

Also, the `$PATH` variable manipulation in `setup.sh` should be done as part of the Go installation process. I think this means we can get rid of `setup.sh` altogether